File: Webgui/web/js/model/rekeningNummer.js
define(['knockout'],
    function (ko) {

        return function () {
            var _this = this;

            _this.id = ko.observable();
            _this.rekeningnummer = ko.observable(),
                _this.bic = ko.observable()
            _this.soortEntiteit = ko.observable();
            _this.entiteitId = ko.observable();

            _this.bicTonen = ko.observable(false);
            _this.parentIdentificatie = ko.observable();
            _this.identificatie = ko.observable();
        };
    });

File: Messaging/src/main/java/nl/lakedigital/as/messaging/domain/RekeningNummer.java
package nl.lakedigital.as.messaging.domain;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name = "rekeningnummer")
public class RekeningNummer extends AbstracteEntiteitMetSoortEnId {
    private String bic;
    private String rekeningnummer;

    public RekeningNummer() {
    }

    public RekeningNummer(SoortEntiteit soortEntiteit, Long entiteitId, String identificatie, String bic, String rekeningnummer) {
        super(soortEntiteit, entiteitId, identificatie);
        this.bic = bic;
        this.rekeningnummer = rekeningnummer;
    }

    public String getBic() {
        return bic;
    }

    public void setBic(String bic) {
        this.bic = bic;
    }

    public String getRekeningnummer() {
        return rekeningnummer;
    }

    public void setRekeningnummer(String rekeningnummer) {
        this.rekeningnummer = rekeningnummer;
    }
}